packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.h

```diff
@@ -20,8 +20,8 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)hideInfoWindow;
 - (BOOL)isInfoWindowShown;
 - (void)removeMarker;
-- (UIImage *)addClusterMarkerText:(NSString *)label;
-- (UIImage *)addPriceMarkerText:(NSString *)label;
+- (UIImage *)clusterMarkerImageWithText:(NSString *)label;
+- (UIImage *)priceMarkerImageWithText:(NSString *)label;
```

Comment:
The compiler doesn't care that the argument's name are mismatched between the .h and .m files, but I do :hehe:
```suggestion
- (UIImage *)clusterMarkerImageWithText:(NSString *)text;
- (UIImage *)priceMarkerImageWithText:(NSString *)text;
```